lib/registration.pm

```diff
@@ -46,7 +46,8 @@ our %SLE15_MODULES = (
     productivity => 'Desktop-Productivity',
     legacy       => 'Legacy',
     script       => 'Scripting',
-    serverapp    => 'Server-Applications'
+    serverapp    => 'Server-Applications',
+    containers   => 'Containers'
```

Comment:
Could we use "contm" as the module name? "contm" is already used in SCC_ADDONS to register containers module for SLE12-SPx.

Although we can define new names for SLE 15, we have to map the names carefully in upgrading tests.

Reply:
Thanks for pointing that out, I changed it for the sake of consistency to `contm`.